### ISSUE TITLE
Add internal modifier for POGivenTH

### ISSUE THREAD

**Issue by talbrecht** [talbrecht/pism_pik#2]
Regarding the commits in https://github.com/talbrecht/pism_pik06/commits/ocean_th_06 there are two aspects that should be discussed for the ocean_th model:

1) Introduce a lower limit of the base for grounded cells to avoid excessive melt
2) Introduce a delta_T modifier, that should not go below pressure melting point

**Comment by talbrecht** [comment]
Using the default delta_T modifier causes unreasobale scaling of shelbmassflux!

**Comment by matthiasmengel** [comment]
Can you elaborate a bit on this? I think we talked about this shortly on skype. so the delta_T does not affect the input fields, but the boundary layer temperature directly, therefore bypassing our 3eqn melting ansatz? we should ask constantine on this when we are sure.

**Comment by talbrecht** [comment]
right, it is the order of calculations that matters: the ocean coupler calculates shelbtemp and shelbmassflux and the -delta_T modifier can add an anomaly to the calculated schelfbtemp. This is problematic since the modifier should be added to the input fields or to the shelfbtemp before the calculation of shelfbmassflux (second option has been realized in pism-pik06). And, shelfbtemp should be bounded below by the pressure melting point...

**Comment by matthiasmengel** [comment]
I think this a general design question then. Valid for other ocean models to come. We should post this on the main issue tracker, shouldn't we?

**Comment by talbrecht** [comment]
you mean in pism dev as a wish? Sure
